src/components/academy/grading/index.tsx

```diff
@@ -242,6 +284,34 @@ class Grading extends React.Component<IGradingProps, State> {
     }
     this.gridApi.exportDataAsCsv({ allColumns: true });
   };
+
+  private handleUnsubmitAlertClose = () => this.setState({ unsubmitAlert: null });
+
+  private unsubmitButton = (props: GradingNavLinkProps) => {
```

Comment:
I feel this state change is rather unnecessary. It is adding unnecessary verbosity to your code and the `null` change feels hacky. Possible to look into a cleaner alternative?

Reply:
Removed, isolated the unsubmit cell component